src/internal/pachd/builder.go

```diff
@@ -548,6 +556,64 @@ func (b *builder) startDebugWorker(ctx context.Context) error {
 	return nil
 }
 
+func (b *builder) generatePJSWorkerAuthToken(ctx context.Context) error {
+	return generatePJSWorkerAuthToken(ctx, b.env.AuthServer(), b.env.GetKubeClient(), b.env.Config())
+}
+
+// TODO: move this function to the auth package(s) it can potentially be called again when we turn on auth.
+func generatePJSWorkerAuthToken(ctx context.Context, authServer auth.APIServer, kubeClient kubernetes.Interface,
+	config *pachconfig.Configuration) error {
+	secretName := "pachyderm-pjs-worker-authentication"
```

Comment:
When we decide where we want this function to live, these ought to be exported constants.

Reply:
Potentially user-configurable?